server/game/gameSystems/GameSystemLibrary.ts

```diff
@@ -146,12 +146,15 @@ export function lookAt<TContext extends AbilityContext = AbilityContext>(propert
 export function moveCard<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IMoveCardProperties, TContext>): CardTargetSystem<TContext> {
     return new MoveCardSystem<TContext>(propertyFactory);
 }
-// /**
-//  * default resetOnCancel = false
-//  */
-// export function playCard(propertyFactory: PropsFactory<PlayCardProperties> = {}): GameSystem {
-//     return new PlayCardAction(propertyFactory);
-// }
+/**
+ * default resetOnCancel = false
+ */
+export function playCardFromHand<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<Omit<IPlayCardProperties, 'playType' | 'optional'>, TContext> = {}): PlayCardSystem<TContext> {
+    // TODO: implement playing with smuggle and from non-standard zones(discard(e.g. Palpatine's Return), top of deck(e.g. Ezra Bridger), etc.) as part of abilties with another function(s)
+    // TODO: implement a "nested" property in PlayCardSystem that controls whether triggered abilities triggered by playing the card resolve after that card play or after the whole ability
+    // playType automatically defaults to PlayFromHand
```

Comment:
Can we explicitly set the value of the `playType` property to `PlayFromHand` using Object.assign so that we're not taking a dependency on this remaining the default

Reply:
That will require checking if `propertyFactory` is a function and altering it differently depending, right?

Reply:
Oh yeah fair, that's more work than it's worth